File: app/src/main/java/com/rdev/tryp/model/login_response/VerifySmsResponse.java
package com.rdev.tryp.model.login_response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class VerifySmsResponse {

    @SerializedName("data")
    @Expose
    private Data data;

    public Data getData() {
        return data;
    }

    public void setData(Data data) {
        this.data = data;
    }

    @Override
    public String toString(){
        return
                "VerifySmsResponse{" +
                        "data = '" + data + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/rdev/tryp/model/login_response/VerifySmsResponse.kt
package com.rdev.tryp.model.login_response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class VerifySmsResponse {

    @SerializedName("data")
    @Expose
    var data: Data? = null

    override fun toString(): String {
        return "VerifySmsResponse{" +
                "data = '" + data + '\''.toString() +
                "}"
    }
    
}